scripts/commands.js

```diff
@@ -18,6 +20,17 @@ exports.study = async (message, args, guildRoles) => {
     await message.delete()
 }
 
+exports.stop = async (message, args, guildRoles) => {
+    guildRoles.forEach(r => {
+        r.role.name === 'Studying' ? message.member.roles.remove(r.role.id) : ''
```

Comment:
Why use a one-line if statement? It clutters the code - same for a few other places - I'd recommend anywhere where you don't need the else, definitely don't use it, and even where you do need the else, consider if a traditional if-statement would be clearer